.gitignore

```
@@ -4,6 +4,7 @@ dist
 build
 .env
 coverage
+upload
```

The folder name should be "upload**s**" here
```suggestion
uploads
```